packages/discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -382,9 +397,9 @@ def save_new_records(
             if (
                 record_type in original_records
                 and entity_id in original_records[record_type]
-                and "is_current"
-                in get_record_columns(original_records[record_type][entity_id])
-                and original_records[record_type][entity_id].is_current
+                and ("is_current"
+                not in get_record_columns(original_records[record_type][entity_id])
```

Comment:
we might be able to get rid of this right?
```
                and ("is_current"
                not in get_record_columns(original_records[record_type][entity_id])
                or original_records[record_type][entity_id].is_current)

```

Reply:
yeah i think we can get rid of it altogether but wanted to do it in a separate pr just in case